src/Workspaces/Core/Portable/Workspace/Solution/StateChecksums.cs

```diff
@@ -70,7 +70,7 @@ public static SolutionStateChecksums Deserialize(ObjectReader reader)
     }
 
     public async Task FindAsync(
-        SolutionState state,
+        SolutionCompilationState compilationState,
```

Comment:
At some point do you imagine we might have two FindAsync() methods, one that handles just the SolutionState for the run-generator case (since we're not going to sync generators over) and one that takes the SolutionCompiltionState that delegates to the former?

Reply:
Yes.  That's exactly how i see it.

Reply:
Yes.  That's precisely what i want to do.